packages/mobile/package.json

```
@@ -34,9 +34,11 @@
     "react-i18next": "^11.18.3",
     "react-native": "0.69.3",
     "react-native-config": "1.4.6",
+    "react-native-linear-gradient": "2.6.2",
     "react-native-mmkv": "2.4.3",
     "react-native-safe-area-context": "4.3.1",
     "react-native-screens": "3.15.0",
+    "react-native-svg": "^12.4.4",
```

Avoid any modifiers please
```suggestion
    "react-native-svg": "12.4.4",
```

---

corrected